crashsimilarity/crash_similarity.py

```diff
@@ -14,39 +14,22 @@
 
 from crashsimilarity import utils
 from crashsimilarity.downloader import SocorroDownloader
+from crashsimilarity.utils import StackTraceProcessor
 
 pyximport.install()
 
 
-def clean_func(func):
-    func = func.lower().replace('\n', '')
-
-    if '@0x' in func:
-        return func[:func.index('@0x') + 3]
-
-    return func
-
-
-def preprocess(stack_trace):
-    return [clean_func(f) for f in stack_trace.split(' | ')][:10]  # XXX: 10 bottom frames or all of them?
-
-
-# Exclude stack traces without symbols.
-def should_skip(stack_trace):
-    return 'xul.dll@' in stack_trace or 'XUL@' in stack_trace or 'libxul.so@' in stack_trace
-
-
 def read_corpus(fnames):
```

Comment:
No problem! Think as if somebody was reading your code in two years from now with little context :)